lib/App.js

```diff
@@ -176,34 +176,74 @@ App.prototype.loadViews = function() {};
 
 App.prototype.loadStyles = function() {};
 
+// This function is defined by requiring 'derby/parsing'
+App.prototype.addViews = null;
+
 App.prototype.component = function(viewName, constructor) {
   if (typeof viewName === 'function') {
     constructor = viewName;
     viewName = null;
   }
-  // DEPRECATED: constructor.prototype.view and constructor.prototype.name
-  var viewFilename = constructor.view || constructor.prototype.view;
-  var constructorIs = constructor.is || constructor.prototype.name;
+
+  var viewArg = constructor.view;
```

Comment:
To me, the name `viewArg` indicates that it's a function argument, when it's actually a property on the constructor.

Some ideas for alternatives: `viewProp`, `viewSpec`

Reply:
I'm cool with viewProp